src/constants/global.constants.ts

```diff
@@ -4,6 +4,7 @@ export const isDevelopment = import.meta.env.VITE_NODE_ENV === "development";
 export const isProduction = import.meta.env.VITE_NODE_ENV === "production";
 export const isAuthEnabled: boolean = import.meta.env.VITE_AUTH_ENABLED === "true";
 export const descopeProjectId: string = import.meta.env.VITE_DESCOPE_PROJECT_ID;
+export const googleAnalytics4Id: string = import.meta.env.GOOGLE_ANALYTICS_ID;
```

Comment:
```suggestion
export const googleAnalyticsId: string = import.meta.env.GOOGLE_ANALYTICS_ID;
```

Reply:
Basically we are talking about Google Analytics 4 - GA4, but yeah, we can do it without the 4 as well.